iogt/templates/base.html

```diff
@@ -8,21 +8,19 @@
         <meta charset="utf-8" />
         <title>{% block title %}{% if self.seo_title %}{{ self.seo_title }}{% else %}{{ self.title }}{% endif %}{% endblock %}{% block title_suffix %}{% endblock %}</title>
         <meta http-equiv="X-UA-Compatible" content="IE=edge" />
-        <meta name="description" content="" />
-        <meta name="keywords" content="" />
-
+        <meta name="description" content="{% if self.social_media_description %}{{ self.social_media_description }}{% else %}{{ self.subtitle }}{% endif %}" />
```

Comment:
This works for Sections and Article pages but not the homepage - A static text option is a good idea.

Reply:
yes, I'm requesting a default description from them, will update before landing

Reply:
@Mitso default text + translations to be added in a separate ticket